aiohttp/base_protocol.py

```diff
@@ -16,6 +18,7 @@ def __init__(self, loop: asyncio.AbstractEventLoop) -> None:
         self._reading_paused = False
 
         self.transport = None  # type: Optional[asyncio.Transport]
+        self.closed = self._loop.create_future()  # type: asyncio.Future[Optional[Exception]]  # noqa
```

Comment:
Please move the future into `client_proto.py`.
Server side protocols don't need it.

Reply:
sure, thank. Fixed.